packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -133,9 +121,9 @@ Media.createMediaConnection = (
         receiveVideo: boolean;
         receiveShare: boolean;
       };
-      audioTrack?: MediaStreamTrack;
-      videoTrack?: MediaStreamTrack;
-      shareTrack?: MediaStreamTrack;
+      audioTrack?: LocalTrack;
```

Comment:
why are we calling all of them as localTrack ?  shoudnt it be local-audio-track and video track ?

Reply:
fixed